bot/utils.py

```diff
@@ -1,11 +1,24 @@
+from collections import Counter as _counter
 import re
 
 # https://regex101.com/r/9dIo49/2
 GAME_REGEX = r'/game @([^\s]+) (\d+):(\d+) @([^\s]+)'
 
+
+def add_result(games, player1, player2, score1, score2, date):
+    new_game = {'Date': date.strftime("%Y-%m-%d"),
+                'Winner': player1 if score1 > score2 else player2,
```

Comment:
what about to replace it with `[(user1, score1), (user2, score2)]` + `sort`?

Reply:
and then:
```py
winner, looser = sortedUsers
```

Reply:
OK